prowler/providers/aws/services/acm/acm_certificates_rsa_key_length/acm_certificates_rsa_key_length.py

```diff
@@ -0,0 +1,24 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.acm.acm_client import acm_client
+
+
+class acm_certificates_rsa_key_length(Check):
+    def execute(self):
+        findings = []
+        for certificate in acm_client.certificates:
+            report = Check_Report_AWS(self.metadata())
+            report.region = certificate.region
+            report.resource_id = certificate.id
+            report.resource_details = certificate.name
+            report.resource_arn = certificate.arn
+            report.resource_tags = certificate.tags
+
+            report.status = "PASS"
+            report.status_extended = f"ACM Certificate {certificate.id} for {certificate.name} meet minimum key size requirements."
+
+            if certificate.key_algorithm == "RSA_1024":
```

Comment:
Just a comment about this, we should be able to configure the list of algorithms that are not secure enough for this check to pass.

Reply:
added in commit https://github.com/prowler-cloud/prowler/pull/4551/commits/4d9d8f394557bbe2a554c2ffc8a6046d9baa44e9

Reply:
Maybe I was not clear enough. My point was to be able to configure the list of algorithms using Prowler configuration. You can see how could it be implemented at https://docs.prowler.com/projects/prowler-open-source/en/latest/developer-guide/checks/#using-the-audit-config

Reply:
Thought about it at first but I wasn't sure if that was what you was asking about. Added in commit c8d395423885ae65bdaa552e8eb18f8b8d8c9fba